multinode-demo/fullnode.sh

```diff
@@ -72,32 +72,51 @@ rsync_url() { # adds the 'rsync://` prefix to URLs that need it
   echo "rsync://$url"
 }
 
-setup_vote_account() {
+setup_vote_and_stake_accounts() {
   declare entrypoint_ip=$1
   declare node_keypair_path=$2
   declare vote_keypair_path=$3
-  declare stake=$4
+  declare stake_keypair_path=$4
+  declare stake=$5
 
-  declare node_keypair
-  node_keypair=$($solana_wallet --keypair "$node_keypair_path" address)
+  declare node_id
```

Comment:
Didn't @mvines just change these to `_keypair` for precision? Why change back?
(maybe just a rebase thing?)

Reply:
it was a rebase thing, but....

it's really not correct to call what you get from $(wallet address) a keypair, it's only the pubkey..

Reply:
updated with yet another way of naming it ;)

Reply:
Good point! Should have looked closer. <3 pubkey